dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -69,92 +69,109 @@ and program attributes too.
+updated if any of its tracked entity attributes, enrollments, events or relationships have been
+modified. The `updatedAt` property of the tracked entity reflects the latest time at which any
+of its tracked entity attributes, enrollments, events or relationships were modified. This
+parameter is inclusive, so results with the exact date and time specified will be included in
+the response.
 
 ### `*.parameter.TrackedEntityRequestParams.updatedWithin`
 
-Get tracked entities updated since given ISO-8601 duration.
+Get tracked entities updated since the given ISO-8601 duration. A tracked entity is considered
+updated if any of its tracked entity attributes, enrollments, events or relationships have been
+modified. The `updatedAt` property of the tracked entity reflects the latest time at which any
+of its tracked entity attributes, enrollments, events or relationships were modified.
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentEnrolledAfter`
 
-Get tracked entities with enrollments that were enrolled after given date and time.
-This parameter is inclusive, so results with the exact date and time specified will be included in the response.
+Get tracked entities that have at least one enrollment with an enrollment date after this date.
+This parameter is inclusive, so results with the exact date and time specified will be included
+in the response.
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentEnrolledBefore`
 
-Get tracked entities with enrollments that were enrolled before given date and time.
-This parameter is inclusive, so results with the exact date and time specified will be included in the response.
+Get tracked entities that have at least one enrollment with an enrollment date before this date.
+This parameter is inclusive, so results with the exact date and time specified will be included
+in the response.
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentOccurredAfter`
 
-Get tracked entities with enrollments occurred after given date and time.
-This parameter is inclusive, so results with the exact date and time specified will be included in the response.
+Get tracked entities that have at least one enrollment with an occurred date after this date.
+This parameter is inclusive, so results with the exact date and time specified will be included
+in the response.
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentOccurredBefore`
 
-Get tracked entities with enrollments occurred before given date and time.
-This parameter is inclusive, so results with the exact date and time specified will be included in the response.
+Get tracked entities that have at least one enrollment with an occurred date before this date.
+This parameter is inclusive, so results with the exact date and time specified will be included
+in the response.
 
 ### `*.parameter.TrackedEntityRequestParams.trackedEntityType`
 
+Get tracked entities with the given tracked entity type. Required if no `program` is specified.
```

Comment:
One between `program`, `trackedEntityType` and `trackedEntities` must be defined.
If no `trackedEntityType` or `program` is defined, then a request with `trackedEntities` will pass validation.